examples/py_proto_library/MODULE.bazel

```diff
@@ -20,7 +20,7 @@ python.toolchain(
 use_repo(python, "python_3_9")
 
 # We are using rules_proto to define rules_proto targets to be consumed by py_proto_library.
-bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
+bazel_dep(name = "rules_proto", version = "6.0.0-rc1")
```

Comment:
FYI this won't work, since this version is not published to BCR.

You need to use a `git_override`:
```starlark
git_override(
    module_name = "rules_proto",
    remote = "https://github.com/bazelbuild/rules_proto",
    commit = "f9b0b880d1e10e18daeeb168cef9d0f8316fdcb5",
)
```

Reply:
bcr is currently stuck. i'll get that fixed and cut a new rc.